New Template "create-lattes-cv"

philkleer opened the pull request:
I am submitting
- [x] a new package/template
- [ ] an update for a package

Description: The template creates from the exported data from [LATTES](https://lattes.cnpq.br/) a CV in the format and with the information needed in regard to standards in Brazil (CNPq). Exports on the website create only mal-formatted RTFs, and, therefore based on a transformation from the XML-File to TOML, this template creates a nice looking curriculum. A vast majority of options is already included, however, [LATTES](https://lattes.cnpq.br/) offers a variety of entries and not all is covered yet. However, for standard curriculums the template should already fit. 

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE
- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

elegaanz commented on packages/preview/create-lattes-cv/0.1.0/lib.typ:
```diff
@@ -0,0 +1,140 @@
+  body,
+) = {
+    // define details:
+    let details = toml(database)
```
Because files are read relatively to the current package/project when using `toml` (and other functions that read from external files), this won't work as expected: you would only be able to access files in the package directory, not in the user's project. You should take the parsed data directly as a parameter, instead of a file path.

elegaanz commented on packages/preview/create-lattes-cv/0.1.0/lib.typ:
```diff
@@ -0,0 +1,140 @@
+  database: "output.toml",
+  kind: "completo", 
+  me: str,
+  last_page: true,
```
As suggested by the package review bot, it would be better if this parameter could use `kebab-case` instead of `snake_case`. And there are a few other occurrences of this in your package.

elegaanz commented on packages/preview/create-lattes-cv/0.1.0/template/example.typ:
I think it would be nice to rename this file to `main.typ` or `cv.typ`, as it will be the name of the file when creating a project from your template.

elegaanz commented:
Also, can you please change the name of the package to something less canonical for what it does?

philkleer commented:
Many thanks for the comments and hints. I solved the problems and also corrected kebab-case and the entry for the toml-file. I changed the template-name to a less canonical one.

elegaanz commented:
Thank you :)
